lib/data/vehicle.rb

```diff
@@ -28,7 +28,8 @@ def self.add_conditions_to_join(join_query, params)
       end
 
       def self.get_random_vehicle
-        ::Vinbot::Data::Vehicle.order(Sequel.lit('RANDOM()')).limit(1).first
+        count = ::Vinbot::Data::Vehicle.count
+        ::Vinbot::Data::Vehicle.where(id: rand(count)).first
```

Comment:
@rrosenblum your concerns are absolutely valid.  However, the database is coming from an another gem so unless the user keeps a local copy of the db and manipulates it, i think the chances of deleted records are slim.